<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deviceConfirmWizardPage</class>
 <widget class="QWizardPage" name="deviceConfirmWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	font: 18pt DinPRO-medium;
	color: white;
}

/****************
 *         v3.0            *
 ****************/
QLabel#titlebarLabel {
	background: rgb( 51, 51, 51 );
}
QLabel#titlebarTextLabel {
    font: 24pt &quot;DINPro-Medium&quot;;
    color: white;
}

QLabel#instructionsLabel
{
    background: rgb( 191, 191, 191 );
}

QLabel#instructionsTextLabel
{
    font: 12pt &quot;DINPro-Medium&quot;;
}</string>
  </property>
  <widget class="QLabel" name="confirmationLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>380</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="deviceLabel">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>90</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="text">
    <string>Device Icon will show here</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="disclaimerLabel">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>330</y>
     <width>551</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>CAUTION: Investigational Device. Limited by United States Law to investigational use only. Exclusively for Clinical Investigations.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
